src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -310,7 +310,7 @@ private void InitCommonPackageBuilderProperties(PackageBuilder builder)
                 builder.MinClientVersion = _packArgs.MinClientVersion;
             }
 
-            CheckForUnsupportedFrameworks(builder);
+            CheckForBadFrameworks(builder);
```

Comment:
This file has no functional changes right?
Can we revert it?

Reply:
oh! Yes, it's a straggler that i forgot to remove. My b 👍🏻